### PR TITLE
Solace plugin

### PR DESCRIPTION
**Pull request by pltod2**

### CODE REVIEW

**Comment by cwiechmann** [comment]
Hi @pltod2, is this supposed to become a real plugin?
I understood in the demo you gave this is some kind of PoC.
If yes, we should add some documentation about the use-case, what Solace is, how to use it, etc.

At the moment it is very bare. Just let me know.

Chris

**Comment by pltod2** [comment]
Hi @cwiechmann, I believe the expectations are this will become a real plugin. But you are right this is WIP. It works for the publish and subscribe cases I was demonstrating but yes don't merge it yet. Probably, I should have flagged it differently or shouldn't submitted the PR at all at this stage. Anyway, for the moment I renamed the PR as WIP, will add test suite and will polish it and then will come back to you for review.

**Comment by cwiechmann** [comment]
I flagged this PR as a draft.

**Comment by cwiechmann** [comment]
What about this PR @pltod2, should we close this or what is the status?

File: api-builder-plugin-solace/README.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+1. Follow the [Getting Started Guide](https://docs.axway.com/bundle/API_Builder_4x_allOS_en/page/api_builder_getting_started_guide.html) to create an API Builder service
+1. Follow the [Axway Flow SDK](https://docs.axway.com/bundle/API_Builder_4x_allOS_en/page/axway_flow_sdk.html) guide to get started creating flow-node plugins.
+
+## Publish
+
+After developing this plugin you can publish it to [npm](https://www.npmjs.com) to make it public.
+
+## Install
+
+After creating your API Builder service (`api-builder init`), you can install this plugin using npm:
```
Comment (cwiechmann) [inline api-builder-plugin-solace/README.md:20]: gdfsfgsdf

File: api-builder-plugin-solace/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+flow editor that is a low-code / no-code solution to designing and developing services
+that integrate to many different connected components, such as databases and APIs.
+
+## Getting started
+
+1. Follow the [Getting Started Guide](https://docs.axway.com/bundle/API_Builder_4x_allOS_en/page/api_builder_getting_started_guide.html) to create an API Builder service
+1. Follow the [Axway Flow SDK](https://docs.axway.com/bundle/API_Builder_4x_allOS_en/page/axway_flow_sdk.html) guide to get started creating flow-node plugins.
+
+## Publish
+
```
Comment (cwiechmann) [inline api-builder-plugin-solace/README.md:15]: Test line 1

File: api-builder-plugin-solace/config/solace.default.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+module.exports = {
+	pluginConfig: {
+		'api-builder-plugin-solace': {			
+			url:      process.env.SOLACE_URL,
+			vpnName:  process.env.SOLACE_VPN,
```
Comment (cwiechmann) [inline api-builder-plugin-solace/config/solace.default.js:5]: Test

**Review by cwiechmann — CHANGES_REQUESTED** [review]
Test

**Comment by pltod2** [comment]
@cwiechmann sorry, did not see your previous comment. Yes, we can close and I am closing this since we have released this as official plugin.

**Pull request closed without merging**